### PR TITLE
Create Zerion trades abstraction

### PR DESCRIPTION
**Pull request by hildobby**
I've checked that:

* [X] the query produces the intended results
* [X] the folder name matches the schema name
* [X] the schema name exists in Dune
* [X] views are prefixed with `view_`, functions with `fn_`.
* [X] the filename matches the defined view, table or function and ends with .sql
* [X] each file has only one view, table or function defined  
* [X] column names are `lowercase_snake_cased`

### CODE REVIEW

File: ethereum/zerion/view_trades.sql [hunk 1]
```diff
@@ -0,0 +1,524 @@
+CREATE TABLE IF NOT EXISTS zerion.view_trades (
```
[the remark this answers is not in the document]
Comment (hildobby) [reply ethereum/zerion/view_trades.sql]: Alright, I removed the 'view_' prefix everywhere

[the remark this answers is not in the document]
Comment (hildobby) [reply ethereum/zerion/view_trades.sql]: Alright, I removed the 'view_' prefix everywhere

File: ethereum/zerion/view_trades.sql [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+      , NULL::decimal AS sold_token_amount
+      , NULL::decimal AS bought_token_amount
+      , NULL::bytea AS sold_token_address
+      , NULL::bytea AS bought_token_address
+      , NULL::text AS sold_token_symbol
+      , NULL::text AS bought_token_symbol
+      FROM transfers_with_prices
+      )
+
+  INSERT INTO dune_user_generated.zerion_trades_reworked
```
[the remark this answers is not in the document]
Comment (hildobby) [reply ethereum/zerion/view_trades.sql]: That's a remnant of my testing, fixed this in my latest commit

**Review by jeff-dude — APPROVED** [review]
@hildobby this should be all built 👍 i pushed a few changes but nothing major. let me know here or discord if anything seems off

**Pull request merged**